packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -715,7 +722,9 @@ export function usePress(props: PressHookProps): PressResult {
             shiftKey: false,
             ctrlKey: false,
             metaKey: false,
-            altKey: false
+            altKey: false,
+            clientX: 0,
+            clientY: 0
```

Comment:
will this come through as 0,0 or will it come through as the center of the target?

Reply:
yeah, i think that the center makes more sense than 0 - rect.left